src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -654,13 +654,13 @@
     <value>Session configuration "{0}" was not found.</value>
   </data>
   <data name="CSCmdsShellNotPowerShellBased" xml:space="preserve">
-    <value>Session configuration "{0}" is not a Windows PowerShell-based shell.</value>
+    <value>Session configuration "{0}" is not a PowerShell-based shell.</value>
 </data>
   <data name="CSCmdsPowerShellCoreShellNotModifiable" xml:space="preserve">
-    <value>Session configuration "{0}" is a PowerShell Core-based shell. Please use PowerShell Core to modify it.</value>
+    <value>Session configuration "{0}" is a PowerShell-based shell. Please use PowerShell to modify it.</value>
```

Comment:
This error seems Core specific, please revert

Reply:
Fixed.